bin/maya.bat

```diff
@@ -23,9 +23,11 @@ if "%MINDBENDER_ASSET%"=="" goto :missing_set_asset
 :: If user forgets to include task with "maya"..
 if "%1"=="" goto :missing_task
 
-set MINDBENDER_WORKDIR=%CD%\work\%1\%USERNAME%\maya
+if not exist "%1" goto :missing_taskfolder
+
+set MINDBENDER_WORKDIR=%CD%\%1\%USERNAME%\maya
 if Not exist %MINDBENDER_WORKDIR% (
-	echo Creating new task "%1"..
+	echo Creating folder for task "%1"..
```

Comment:
I wouldn't worry about that, as the person using `new` - like Calle - won't be the same as the one using the created bat files, like Derek.

But being more specific is a good idea, in this case we aren't creating a new task, we are entering a task for the first time. If Calle has created `maya modeling` but Derek hasn't modeled before, we should tell Derek that this will be the first time.